lib/assert.js

```diff
@@ -344,11 +344,17 @@ assert.notStrictEqual = function notStrictEqual(actual, expected, message) {
   }
 };
 
-function createMsg(msg, key, actual, expected) {
-  if (msg)
-    return msg;
-  return `${key}: expected ${inspect(expected[key])}, ` +
-    `not ${inspect(actual[key])}`;
+function compareKey(actual, expected, key, msg) {
```

Comment:
Nit: could we give this function a slightly more descriptive name since it only works for `throws`? Like `compareExceptionKey` or something?

Reply:
Done